test/db.test.ts

```diff
@@ -0,0 +1,18 @@
+import initiate from "../src/db/db";
+
+describe("Database Connection", () => {
+	it("should connect to database", async () => {
```

Comment:
Use a simpler test like true === true or something.

Reply:
instead of this database test, I should have a test where it expects true to be true.

@IyiKuyoro